randovania/data/json_data/prime1.txt

```diff
@@ -42,8 +45,6 @@ Crater Entry Point
 Asset id: 2472970646
 > Dock to Crater Tunnel A; Heals? False
   * Plasma Door to Crater Tunnel A/Dock to Crater Entry Point
-  > Teleport to Artifact Temple
-      Trivial
 
 > Teleport to Artifact Temple; Heals? False; Spawn Point
   * Teleporter to Tallon Overworld - Artifact Temple
```

Comment:
This room is missing a Save Station node. The teleporter and door should connect to the save as trivial, and the save to to the door and teleporter as trivial.

Reply:
I completely forgot about this one I'll fix it

Reply:
Crater Tunnel A is still missing a connection to somewhere

Reply:
oops forgot about it sorry